scripts/tests/linux/log_line_processing.py

```diff
@@ -92,6 +96,7 @@ def __enter__(self):
             stdout=subprocess.PIPE,
             stderr=subprocess.PIPE,
             text=True,
+            bufsize=1,
```

Comment:
add a comment this meaning line buffering and explaining why this is correct.

Reply:
I've added a comment in the latest commit.